### PR TITLE
Fix Policy Injection

### PR DESCRIPTION
**Pull request by pablitoc**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -74,6 +74,7 @@ function create_ecr_repo() {
       if echo ${output} | grep -q RepositoryNotFoundException; then
         echo "== REPO DOESN'T EXIST, CREATING.."
         aws ecr create-repository --region $AWS_DEFAULT_REGION --repository-name $INPUT_REPO
+        aws ecr set-repository-policy --region $AWS_DEFAULT_REGION --repository-name $INPUT_REPO --policy-text file://"ecr_policy.json"
```
Comment (mhatch-nxcr) [inline entrypoint.sh]: where is `ecr_policy.json` coming from?

Comment (pablitoc) [reply entrypoint.sh]: im actually reverting that, but will be in each repo

Comment (grante-nxcr) [reply entrypoint.sh]: it is another file in the repository at the same filesystem level as the shell script

**Comment by grante-nxcr** [comment]
What is the point of moving the repo policy setup to a shell script vs having it in the action config? Was there something not working about the config setup? (I assume that was the case based on the branch name but just curious the details)

**Comment by pablitoc** [comment]
passing json in command line is ugly. CLI is easier when you pass in the file. I didnt want to keep it in the repo because this is a public repo.

**Review by grante-nxcr — APPROVED** [review]

**Comment by pablitoc** [comment]
@mhatch-nxcr @grante-nxcr this is ready for merging. Here is an example of how it should look like in the workflow: https://github.com/nxcr-org/test-repo/blob/main/.github/workflows/cd.yml#L19-L32

**Review by mhatch-nxcr — APPROVED** [review]

**Pull request closed without merging**